server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/Authentication.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 9, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @author Daniel M. Zimmerman <dmz@galois.com>
+ * @description A system to assist in conducting statewide risk-limiting audits.
+ */
+
+package us.freeandfair.corla.endpoint;
+
+import javax.persistence.PersistenceException;
+
+import spark.Request;
+
+import us.freeandfair.corla.Main;
+import us.freeandfair.corla.hibernate.Persistence;
+import us.freeandfair.corla.model.Administrator;
+import us.freeandfair.corla.model.Administrator.AdministratorType;
+import us.freeandfair.corla.model.AdministratorQueries;
+
+/**
+ * Authentication tasks used by many endpoints.
+ * 
+ * @author Daniel M. Zimmerman
+ * @version 0.0.1
+ */
+public final class Authentication {  
+  /**
+   * The constant for the "admin" property of the current session.
+   */
+  public static final String ADMIN = "admin";
+  
+  /**
+   * The constant for the "username" request parameter.
+   */
+  public static final String USERNAME = "username";
+  
+  /**
+   * The constant for the "password" request parameter.
+   */
+  public static final String PASSWORD = "password";
+  
+  /**
+   * The constant for the "second factor" request parameter.
+   */
+  public static final String SECOND_FACTOR = "second_factor";
+  
+  /**
+   * Private constructor to prevent instantiation.
+   */
+  private Authentication() {
+    // do nothing
+  }
+  
+  /**
+   * Checks to see whether the specified request is authenticated as
+   * the specified administrator type.
+   * 
+   * @param the_request The request.
+   * @param the_type The administrator type.
+   * @return true if the request is authenticated, false otherwise.
+   */
+  public static boolean isAuthenticated(final Request the_request, 
```

Comment:
Maybe rename this to `isAuthenticatedAs()`, which would match the `authenticateAs()` method below.